code/modules/power/supermatter/supermatter.dm

```diff
@@ -678,6 +678,18 @@ GLOBAL_DATUM(main_supermatter_engine, /obj/machinery/power/supermatter_crystal)
 		if(produces_gas)
 			env.merge(removed)
 			air_update_turf()
+	if(takes_damage)
+		for(var/turf/turf_to_check in view(1, loc))
+			if(isspaceturf(turf_to_check) && istype(loc))
+				var/integrity = get_integrity()
+				if(integrity < 15)
+					damage += max((power * 0.0001) * DAMAGE_INCREASE_MULTIPLIER, 0)
+				else if(integrity < 25)
+					damage += max((power * 0.001) * DAMAGE_INCREASE_MULTIPLIER, 0)
+				else if(integrity < 45)
+					damage += max((power * 0.05) * DAMAGE_INCREASE_MULTIPLIER, 0)
```

Comment:
has this been tested? this seems a bit overtuned and seemingly comes out of nowhere, a sudden damage spike randomly between 25 and 45 integrity?

for reference the damage ratio from this to above 45 is 0.05/0.0001 which is 500/1, meaning for every damage point from 45 to 100 integrity results in 500 damage points here.

Reply:
What? What's that math you made?
If you open a hole in space and let's say the SM power is around 1500:
If the integrity is over 45% you get 1500*0.0001*0.25 resulting in 0.04 points of damage.
And if the integrity is less than 45% but over 25% you get 1500*0.05*0.25 that is around 20 points of damage, that compared to the amount of health the crystal have is nothing! Also the damage will slow down the closer to 0% you are, giving the players the "scare" of having a very fast delamination that slow down at the end

Reply:
so if the damage is minimal then why is this still not viable

also you can multiply the damage by having extra turfs removed in view

Reply:
i could add a break to check only one turf, but i want to wait to see this on live